pkg/kev/skaffold.go

```diff
@@ -262,6 +284,88 @@ func (s *SkaffoldManifest) AdditionalProfiles() {
 	}
 }
 
+// SetBuildArtifacts detects build artifacts from the current project and adds `build` section to the manifest
+func (s *SkaffoldManifest) SetBuildArtifacts(analysis *Analysis) error {
+
+	// don't set build artefacts if no analysis available
+	if analysis == nil {
+		return nil
+	}
+
+	buildArtifacts := map[string]string{}
+
+	for _, d := range analysis.Dockerfiles {
```

Comment:
Would be cool if this block is factored out into its own func `e.g. collectImagesArtefacts(...) or something similar`. 

Helps communicate what's happening without having to trawl through the code.